airbyte-integrations/connectors/source-google-search-console/credentials/credentials.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this file should be in a dir called `secrets/` otherwise creds will get checked into source control by accident